tests/DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Faker/Fakers/GiasGroupLinkFaker.cs

```diff
@@ -0,0 +1,30 @@
+using System.Globalization;
+using Bogus;
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Models.Gias;
+
+namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Faker.Fakers;
+
+public class GiasGroupLinkFaker
+{
+    private readonly Faker<GiasGroupLink> _groupLinkFaker;
+    private readonly DateTime _refDate;
+
+    public GiasGroupLinkFaker(DateTime refDate)
+    {
+        _groupLinkFaker = new Faker<GiasGroupLink>("en_GB");
+        _refDate = refDate;
+    }
+
+    public GiasGroupLink Generate(string uid, string joinedDate, string urn)
```

Comment:
The parameter `joinedDate` is actually the group open date, we should rename this parameter